tests/integration/docker_images/popular-containers-images.bats

```diff
@@ -289,7 +290,7 @@ setup() {
 }
 
 @test "start instance in percona container" {
-	if $DOCKER_EXE run -i percona perl -e 'print "Clear Containers\n"' | grep LANG; then false; else true; fi
+	if $DOCKER_EXE run --rm -i percona perl -e 'print "Clear Containers\n"' | grep LANG; then false; else true; fi
```

Comment:
This construct "if xxx | grep foo; then false; else true; fi" could have been done with "if xxx | grep -v foo" (no need to do anything about this, for next time you're around that code).